jobs/build_docker/build_docker.sh

```diff
@@ -1,41 +1,45 @@
-#!/bin/bash
+#!/bin/bash -ex
 #download manifest
 curl --user $BINTRAY_CREDS -L "$MANIFEST_FILE_URL" -o rackhd-manifest
 
 #clone
 ./build-config/build-release-tools/HWIMO-BUILD ./build-config/build-release-tools/application/reprove.py \
-  --manifest rackhd-manifest \
-  --builddir ./$CLONE_DIR \
-  --jobs 8 \
-  --force \
-  checkout \
-  packagerefs-commit
+--manifest rackhd-manifest \
+--builddir ./$CLONE_DIR \
+--jobs 8 \
+--force \
+checkout \
+packagerefs-commit
 
 rsync -r ./$CLONE_DIR/RackHD/ ./build/
 
-#build static files
-pushd ./$CLONE_DIR/on-imagebuilder
+on_imagebuilder_version=$( ./build-config/build-release-tools/HWIMO-BUILD ./build-config/build-release-tools/application/version_generator.py \
+--repo-dir ./$CLONE_DIR/on-imagebuilder \
+--is-official-release $IS_OFFICIAL_RELEASE )
 
-echo $SUDO_PASSWORD |sudo -S ./build_all.sh
+common_path=/var/renasar/on-http/static/http/common
+pxe_path=/var/renasar/on-tftp/static/tftp
 
-#copy to on-imagebuilder folder for docker build
-output_path=/tmp/on-imagebuilder
+echo $SUDO_PASSWORD |sudo -S rm -rf /etc/apt/sources.list.d/rackhd.source.list
+echo "deb https://dl.bintray.com/rackhd/debian trusty main" | sudo tee -a /etc/apt/sources.list.d/rackhd.source.list
```

Comment:
I think this should be rackhd-mirror, this is the cached version you need. build from previous "debian build" step.